packages/components/CHANGELOG.md

```diff
@@ -8,6 +8,54 @@ menu: Changelog
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [3.0.0-pre.8](https://github.com/GetJobber/atlantis/compare/@jobber/components@2.90.0...@jobber/components@3.0.0-pre.8) (2023-02-01)
```

Comment:
Oooh how did this change show up here? I thought walter does this automatically 🤔 y dis here?